From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Oleg <balbes-150@yandex.ru>
Date: Tue, 11 Jan 2022 15:26:11 +0100
Subject: [ARCHEOLOGY] move kernel edge to 5.16 (#3387)

> X-Git-Archeology: - Revision 6b490e16944b30ff69bf9c13678905187df0d9d4: https://github.com/armbian/build/commit/6b490e16944b30ff69bf9c13678905187df0d9d4
> X-Git-Archeology:   Date: Tue, 11 Jan 2022 15:26:11 +0100
> X-Git-Archeology:   From: Oleg <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: move kernel edge to 5.16 (#3387)
> X-Git-Archeology:
> X-Git-Archeology: - Revision f52a4193d02ef88333ba117c68d49486dfd7ff41: https://github.com/armbian/build/commit/f52a4193d02ef88333ba117c68d49486dfd7ff41
> X-Git-Archeology:   Date: Sun, 20 Mar 2022 22:58:21 +0100
> X-Git-Archeology:   From: Oleg <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: Adding Pine64 Quartz64a as WIP target (#3539)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 0afe24c95729044910e0b3f84dc5500bcdc6524c: https://github.com/armbian/build/commit/0afe24c95729044910e0b3f84dc5500bcdc6524c
> X-Git-Archeology:   Date: Sun, 24 Apr 2022 22:33:47 +0200
> X-Git-Archeology:   From: Oleg <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: move kernel media edge to 5.17 (#3704)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 2e1b4eed739d6ea81c3a7ba7e94d1bcab5cbbb6c: https://github.com/armbian/build/commit/2e1b4eed739d6ea81c3a7ba7e94d1bcab5cbbb6c
> X-Git-Archeology:   Date: Mon, 04 Jul 2022 16:18:31 +0300
> X-Git-Archeology:   From: balbes150 <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: move kernel media to current 5.18 and edge 5.19
> X-Git-Archeology:
> X-Git-Archeology: - Revision 738c1613073bfd504aebc223a0d110cc6afefbe5: https://github.com/armbian/build/commit/738c1613073bfd504aebc223a0d110cc6afefbe5
> X-Git-Archeology:   Date: Sat, 12 Nov 2022 10:11:22 +0100
> X-Git-Archeology:   From: Oleg <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: move kernel media-current to 5.19.17 (#4406)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 276f346ee9c21cdcd627e54676334f4d5a9292b6: https://github.com/armbian/build/commit/276f346ee9c21cdcd627e54676334f4d5a9292b6
> X-Git-Archeology:   Date: Wed, 14 Dec 2022 10:17:09 +0100
> X-Git-Archeology:   From: Oleg <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: move kernel media to current 6.0 and edge 6.1 (#4566)
> X-Git-Archeology:
---
 arch/arm64/boot/dts/rockchip/rk3328-roc-pc.dts | 16 +++++++++-
 1 file changed, 15 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3328-roc-pc.dts b/arch/arm64/boot/dts/rockchip/rk3328-roc-pc.dts
index e3e3984d01d4..aa3857535455 100644
--- a/arch/arm64/boot/dts/rockchip/rk3328-roc-pc.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3328-roc-pc.dts
@@ -28,7 +28,7 @@ button-recovery {
 	ir-receiver {
 		compatible = "gpio-ir-receiver";
 		gpios = <&gpio2 RK_PA2 GPIO_ACTIVE_LOW>;
-		linux,rc-map-name = "rc-khadas";
+		linux,rc-map-name = "rc-station";
 		pinctrl-names = "default";
 		pinctrl-0 = <&ir_int>;
 	};
@@ -81,6 +81,20 @@ bt_en: bt-en {
 	};
 };
 
+&sdmmc_ext {
+	bus-width = <4>;
+	cap-sd-highspeed;
+	cap-sdio-irq;
+	keep-power-in-suspend;
+	mmc-pwrseq = <&sdio_pwrseq>;
+	non-removable;
+	num-slots = <1>;
+	pinctrl-names = "default";
+	pinctrl-0 = <&sdmmc0ext_bus4 &sdmmc0ext_cmd &sdmmc0ext_clk>;
+	sd-uhs-sdr104;
+	status = "okay";
+};
+
 &pmic_int_l {
 	rockchip,pins = <0 RK_PA2 RK_FUNC_GPIO &pcfg_pull_up>;
 };
-- 
Armbian

